src/test/run-selftest-pg

```diff
@@ -30,6 +29,18 @@ PSQL=psql
 findpg pg_ctl
 PGCTL=pg_ctl
 
+if [ "$(which $PGCTL)" == "" ]; then
+    findpg pg_ctlcluster
+    PGCTL=pg_ctlcluster
+
+    if [ "$(which $PGCTL)" == "" ]; then
+        echo "Could not find pg_ctl or pg_ctlcluster"
+        exit 1
+    fi
+fi
+
+exit
```

Comment:
I think this `exit` was added by accident

Reply:
:(

Testing the tests. Yep, removed. Thanks for the pickup.